src/functions-and-arrays.js

```diff
@@ -1,38 +1,88 @@
 // Iteration #1: Find the maximum
-function maxOfTwoNumbers() {}
+
+function maxOfTwoNumbers(x,y) {
+  if(x>=y){
```

Comment:
in this case we should name at number1 number2 or anything else except just literals